gdc_maf_tool/gdc_api_client.py

```diff
@@ -177,15 +179,19 @@ def collect_criteria(hit_map: Dict) -> List[PrimaryAliquotSelectionCriterion]:
             PrimaryAliquotSelectionCriterion(
                 id=hit["file_id"],
                 samples=sample_criteria,
-                case_id=hit["case_id"],
+                entity_id=hit[entity],
                 maf_creation_date=hit["created_datetime"],
             )
         )
     return criteria
 
 
 def collect_mafs(
-    project_id: str, case_ids: List[str], file_ids: List[str], token: Optional[str]
+    project_id: str,
+    disable_aliquot_selection: bool,
```

Comment:
Once we rename the cli argument, we'll also need to name this parameter.

Reply:
yep